deployments/Dockerfiles/localnode/tendermint.py

```diff
@@ -189,13 +189,22 @@ def _stop_tm_process(self) -> None:
         if self._process is None:
             return
 
-        if platform.system() == "Windows":  # pragma: nocover
-            os.kill(self._process.pid, signal.CTRL_C_EVENT)  # type: ignore  # pylint: disable=no-member
+        if platform.system() == "Windows":
+            self._process.send_signal(signal.CTRL_C_EVENT)  # type: ignore  # pylint: disable=no-member
         else:
-            os.killpg(os.getpgid(self._process.pid), signal.SIGTERM)
+            self._process.send_signal(signal.SIGTERM)
+
+        self._process.wait(timeout=30)
+        poll = self._process.poll()
+        if poll is None:  # pragma: nocover
+            self._process.terminate()
+            self._process.wait(3)
```

Comment:
Does this mean that we might wait for up to 33 secs? If yes, then this could be an issue if the reset and pause timeout is <= 33 secs. It might be worth mentioning it in the docs or guarding against it.

Also, why 30 and 3? 
Could we create constants for these values?

Reply:
So I addressed this in https://github.com/valory-xyz/open-autonomy/pull/1064/commits/0bf09bd28834b2e5636e1a51da1efe0a45692bc3 , I set the wait for exit timeout to 5 seconds. 30 or 5 and 3 are the values we are also using in some other places, there's no particular reason for them.